src/views/RecIM/components/List/Listing/index.js

```diff
@@ -132,8 +134,17 @@ const ActivityListing = ({ activity }) => {
   );
 };
 
-const TeamListing = ({ team, match, setTargetTeamID }) => {
+const TeamListing = ({ team, invite, match, setTargetTeamID }) => {
   if (!team && !match) return null;
+
+  const handleAcceptInvite = async () => {
+    await respondToTeamInvite(team.ID, 'accepted');
+  };
+
+  const handleRejectInvite = async () => {
+    await respondToTeamInvite(team.ID, 'rejected');
+  };
```

Comment:
My only gripe about this is that we can't easily pass back a closeWithSnackbar unless we pop a snackbar in the listing component, (which seems backwards) this also affects a reload to remove the teamlisting.